src/datumaro/plugins/transforms.py

```diff
@@ -1069,6 +1069,18 @@ def _resize_image():
 
         return _resize_image
 
+    @staticmethod
+    def _lazy_resize_rlemask(mask, new_size):
+        def _resize_image():
+            # Can use only NEAREST for masks,
+            # because we can't have interpolated values
+            rescaled_mask = cv2.resize(
+                mask.image.astype(np.float32), new_size[::-1], interpolation=cv2.INTER_NEAREST
```

Comment:
Is there any special reason to change the `dtype` into `float32`? It requires more computations than `uint8`.

Reply:
Fix in [ca4187b](https://github.com/openvinotoolkit/datumaro/pull/1361/commits/ca4187b91c6e0a698408a00a72288a35712a7d41)